src/amo/components/LandingPage.scss

```diff
@@ -3,3 +3,69 @@
 .LandingPage {
   padding: $padding-page;
 }
+
+.Addon-Header {
+  margin: 10px;
+  max-width: 100%;
+  text-align: center;
+}
+
+.Icon-persona {
+  background: url('~amo/img/icons/themes.svg') 50% 50% no-repeat;
+  background-size: cover;
+  display: inline-block;
+  height: 50px;
+  margin-bottom: 15px;
+  width: 41.5px;
+}
+
+.Icon-extension {
+  background: url('~amo/img/icons/extensions.svg') 50% 50% no-repeat;
+  background-size: cover;
+  display: inline-block;
+  height: 47.8px;
+  margin-bottom: 15px;
+  width: 49px;
+}
+
+.Addon-Header-Text {
+  display: inline-block;
+  margin-left: 20px;
```

Comment:
We need to support RTL as well so anything like `margin-left`, `padding-left` or `left` will need to be flipped for RTL.

There are some helpers you can use in `mixins.scss`. This could instead be `@include margin-start(20px)`.